tests/suites/test_suite_psa_crypto_util.function

```diff
@@ -0,0 +1,44 @@
+/* BEGIN_HEADER */
+#include <stdint.h>
+#include <test/helpers.h>
+#include <mbedtls/psa_util.h>
+#include <mbedtls/asn1write.h>
+
+enum  {
+    ECDSA_RAW_TO_DER = 0,
+    ECDSA_DER_TO_RAW,
+};
+/* END_HEADER */
+
+/* BEGIN_DEPENDENCIES
+ * depends_on:MBEDTLS_PSA_CRYPTO_C:MBEDTLS_ASN1_WRITE_C:MBEDTLS_ASN1_PARSE_C
+ * END_DEPENDENCIES
+ */
+
+/* BEGIN_CASE */
+void ecdsa_raw_to_der(int direction, int key_bits, data_t *input, data_t *exp_result, int exp_ret)
+{
+    unsigned char *tmp_buf = NULL;
+    size_t tmp_buf_len = exp_result->len;
+    size_t ret_len;
+
+    TEST_CALLOC(tmp_buf, tmp_buf_len);
+
+    if (direction == ECDSA_RAW_TO_DER) {
```

Comment:
I think I'd prefer two test functions, one per direction. It would also allow each test function to have the proper ASN.1 dependency (write OR parse but not both).